test/e2e/calico/calico_suite_test.go

```diff
@@ -0,0 +1,269 @@
+package calico
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/manifest"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	f               *framework.Framework
+	err             error
+	uiNamespace     = "management-ui"
+	clientNamespace = "client"
+	starsNamespace  = "stars"
+	uiLabel         = map[string]string{"role": "management-ui"}
+	clientLabel     = map[string]string{"role": "client"}
+	feLabel         = map[string]string{"role": "frontend"}
+	beLabel         = map[string]string{"role": "backend"}
+	uiPod           v1.Pod
+	clientPod       v1.Pod
+	fePod           v1.Pod
+	bePod           v1.Pod
+)
+
+func TestCalicoPoliciesWithVPCCNI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Calico with VPC CNI e2e Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+	f = framework.New(framework.GlobalOptions)
+	By("installing Calico operator")
+
+	tigeraVersion := f.Options.CalicoVersion
+	err := f.InstallationManager.InstallTigeraOperator(tigeraVersion)
+	Expect(err).ToNot(HaveOccurred())
+
+	By("Patching ARM64 node unschedulable")
```

Comment:
Is there a reason we mark it unschedulable?

Reply:
Oh is it because star policy is still not supported for ARM instance types?

Reply:
Cordon these arm nodes to avoid Calico tests pods deployed to them. For now the provided tests containers are for AMD.